### ISSUE TITLE
⌨ Insufficient link context in Admin tables

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#4128]
## Description
Some links in the Admin tables do not provide sufficient context to users that may be accessing the site using assistive technology.

## Screenshots

![Screen Shot 2023-07-05 at 15 07 32](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/a1dbd948-497b-4403-aec7-99df9fbba099)

![Screen Shot 2023-07-05 at 15 09 38](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/485494fa-ee56-4279-bf95-8182125e3de5)

## Acceptance Criteria

- [ ] all links in Admin tables have sufficient context when using assistive technology

**Comment by brindasasi** [comment]
@GCTC-NTGC/designers  could we get the help on descriptive link context ?

**Comment by gobyrne** [comment]
This issue is stale and we've done a fair bit of accessibility testing and fixing since it was created. Closing

**Comment by mnigh** [comment]
@gobyrne The screenshots in this issue's description were taken two weeks ago. The links highlighted still lack context for screen readers.

**Comment by RM-1978** [comment]
Potential solutions might be to standardize tables across the platform as part of #6932

**Comment by mnigh** [comment]
Thanks @RM-1978 and @esizer! I've added your recommendations as acceptance criteria items in the description of #6932 and will close this issue accordingly.
